contribution_guidelines.md

```diff
@@ -0,0 +1,13 @@
+# Contribution guidelines
+
+ContainerCollection is an open source library, we welcome contributors. 
+
+Before you send a PR, however, please make sure your change complies with these quick guidelines:
+
+- Always unit test your code, PR won't be approved without it.
+- Document APIs, in a way that makes new users of those APIs understand how they work, and highlight things that are good to keep in mind when using them.
+- Follow our [style guide](README.md).
+- For larger API change request, it's recommended to open an issue, where you can get quicker feedback on your idea instead of having to wait until it has been implemented or your PR gets approved.
+
+## Code of conduct
+This project adheres to the [Open Code of Conduct](https://github.com/todogroup/opencodeofconduct).  By participating, you are expected to honor this code.
```

Comment:
Maybe we could write our own CoC. Looks like the OCoC is discontinued.

Reply:
Yep, sounds good. Will do that!

Reply:
@peredaniel @LeeMatthewHiggins I've added my 5 cents to the Code of Conduct but any contribution is welcome. Could you please give it a peek? Thanks